spec/authentication_service/request_spec.rb

```diff
@@ -65,7 +65,7 @@
         let(:status) { { status: 401, body: result.to_json } }
 
         it 'raises an UnauthorizedUserError' do
-          expect { subject.run(user) }.to raise_error(Hyper::AuthenticationService::UnauthorizedUserError)
+          expect { subject.run(user) }.to raise_error(Hyper::AuthenticationService::UnauthorizedUserError, result["error"])
```

Comment:
Looks like we don't have rubocop set up for this repo, but strings should be single quoted unless double quotes are necessary